docs/reference/commandline/service_create.md

```diff
@@ -474,6 +474,44 @@ accessible at the target port on every node regardless if there is a task for
 the service running on the node. For more information refer to
 [Use swarm mode routing mesh](https://docs.docker.com/engine/swarm/ingress/).
 
+### Create services using templates
+
+You can use templates for some flags of `service create`, using the syntax
+provided by the Go's [text/template](http://golange.org/pkg/text/template/) package.
+
+The supported flags are the following :
+
+- `--hostname`
+- `--mount`
+- `--env`
+
+Valid placeholders for the Go template are listed below:
+
+Placeholder       | Description
+----------------- | --------------------------------------------
+`.Service.ID`     | Service ID
+`.Service.Name`   | Service name
+`.Service.Labels` | Service labels
+`.Node.ID`        | Node ID
+`.Task.ID`        | Task ID
+`.Task.Name`      | Task name
+`.Task.Slot`      | Task slot
```

Comment:
> Also, probably it would be better to note that multiple tasks can share the single slot number in certain cases

I think you're over-valuing the importance of this particular functionality. It is not important for most users and calling it out every makes it look less useful than it is. In this case, the behavior of `Task.Slot` isn't important, at all.

In addition, the contents of `.Task.Slot` should largely be treated as opaque. This avoids consumers making assumptions about the contents of a field that may be broken in future versions.